src/main/res/values/strings.xml

```diff
@@ -45,4 +45,11 @@
 	<string name="promptChooseImage">Choose image</string>
 
 	<string name="spnCompressingImage">Compressing image…</string>
+
+	<string name="locRequest_title">Location Access</string>
+	<string name="locRequest_message">%s collects location data when you submit a form to analyze and improve health outcomes in your area. Select \"Allow while using the app\" on the next screen to turn on your location access.</string>
+	<string name="locRequest_okButton">Turn on</string>
+	<string name="locRequest_denyButton">No thanks</string>
+    <string name="locRequest_IconDescription">My location icon</string>
+	<string name="locRequest_mapIconDescription">Location in map icon</string>
```

Comment:
I think the mix of snake and camel case for the names is weird. I know there's examples of this above, but let's start using camel case only.

Reply:
Yes, I had the same filling but I was following the pattern. I'll change to camelCase only.

Reply:
Changed all new string definitions to just camelCase.